### PR TITLE
Fix consumer info if consumer was closed

### PR DESCRIPTION
**Pull request by Jarema**
Co-authored-by: Derek Collison <derek@nats.io>
Signed-off-by: Tomasz Pietrek <tomasz@nats.io>
Signed-off-by: Derek Collison <derek@nats.io>

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -17,6 +17,8 @@ github.com/nats-io/jwt/v2 v2.5.2 h1:DhGH+nKt+wIkDxM6qnVSKjokq5t59AZV5HRcFW0zJwU=
 github.com/nats-io/jwt/v2 v2.5.2/go.mod h1:24BeQtRwxRV8ruvC4CojXlx/WQ/VjuwlYiH+vu/+ibI=
 github.com/nats-io/nats.go v1.30.2 h1:aloM0TGpPorZKQhbAkdCzYDj+ZmsJDyeo3Gkbr72NuY=
 github.com/nats-io/nats.go v1.30.2/go.mod h1:dcfhUgmQNN4GJEfIb2f9R7Fow+gzBF4emzDHrVBd5qM=
+github.com/nats-io/nats.go v1.30.2 h1:aloM0TGpPorZKQhbAkdCzYDj+ZmsJDyeo3Gkbr72NuY=
```
Comment (derekcollison) [inline go.sum:20]: Run `go mod tidy`

**Review by derekcollison — CHANGES_REQUESTED** [review]

File: go.mod [hunk 2]
```diff
@@ -18,5 +18,6 @@ require (
 
 require (
 	github.com/golang/protobuf v1.4.2 // indirect
+	golang.org/x/text v0.13.0 // indirect
```
Comment (derekcollison) [inline go.mod:21]: What pulled this in?

Comment (Jarema) [reply go.mod:21]: the new nats.go client `jetstream` module used only in test.

File: go.sum (hunk 1 above)
Comment (Jarema) [reply go.sum:20]: side effect of `jetstream` module usage AFAIK.

**Review by derekcollison — APPROVED** [review]
LGTM

**Pull request merged**